android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -352,6 +352,9 @@ public void resumePreview() {
             @Override
             public void run() {
                 synchronized(this){
+                    //if(mCamera == null){
+                    //    start();
+                    //}
```

Comment:
This can be removed?

Reply:
Sure, sorry, forgot it there after testing. Pushed a updated removing the comments.